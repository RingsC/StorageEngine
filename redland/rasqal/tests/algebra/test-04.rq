# http://www.w3.org/TR/2008/REC-rdf-sparql-query-20080115/#convertGraphPattern
#
# Example: group consisting a union of a union and a basic graph pattern:
#
PREFIX : <http://example.org#>
SELECT * WHERE
{ { ?s :p1 ?v1 } UNION {?s :p2 ?v2 } UNION {?s :p3 ?v3 } }
