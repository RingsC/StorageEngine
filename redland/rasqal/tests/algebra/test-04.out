Project(
        Union(
              Union(
                    BGP(
                        triple(variable(s), uri<http://example.org#p1>, variable(v1))
                    ) ,
                    BGP(
                        triple(variable(s), uri<http://example.org#p2>, variable(v2))
                    )
              ) ,
              BGP(
                  triple(variable(s), uri<http://example.org#p3>, variable(v3))
              )
        ) ,
        Variables([ variable(s), variable(v1), variable(v2), variable(v3) ])
)
